### ISSUE TITLE
Allow grafana install via ansible

### ISSUE THREAD

**Issue by sakoush** [SeldonIO/seldon-core#4696]
Currently ansible automation does not install grafana. It would be useful to have this as part of the process. Things to consider for configuration:
- grafana version
- prometheus datasource
- user settings
- populate our example dashboard [here](https://github.com/SeldonIO/seldon-core/blob/v2/prometheus/dashboards/seldon.json)

I think there is already a way in [kube-prometheus](https://github.com/SeldonIO/seldon-core/tree/v2/prometheus) that could setup grafana.

**Comment by RafalSkolasinski** [comment]
Yes, that will be nice to have a ready to go example to show some features. Still will need to make clear that this is just an example and Grafana installation / configuration is lies out of Seldon responsibility.
